src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => (
+    century === undefined
+      ? person.sex === 'm'
+      : person.sex === 'm' && Math.ceil(person.died / 100) === century
+  ));
+  const calculateAge = men.map(person => person.died - person.born);
+  const calculateAverageAge = calculateAge.reduce((sum, age) => (
+    (sum + age))) / men.length;
```

Comment:
It's better to make a separate function for that purpose in the end of a file, before exports